File: calc.js
const prompt = require("prompt-sync")()

function add(x, y) {
    return x + y;
}

function subtract(x, y) {
    return x - y;
}

function multiply(x, y) {
    return x * y;
}

function divide(x, y) {
    return x / y;
}

console.log("Select operation: ")
console.log("1. Add")
console.log("2. Subtract")
console.log("3. Multiply")
console.log("4. Divide")
let choice = prompt("Select from above: ")
let num1 = parseInt(prompt("Enter the first number: "))
let num2 = parseInt(prompt("Enter the second number: "))

if (choice == "1") {
    console.log("The answer is: ", add(num1, num2))
}

if (choice == "2") {
    console.log("The answer is: ", subtract(num1, num2))
}

if (choice == "3") {
    console.log("The answer is: ", multiply(num1, num2))
}

if (choice == "4") {
    console.log("The answer is: ", divide(num1, num2))
}

if (choice < 1) {
    console.log("Invalid input")
}

if (choice > 4) {
    console.log("Invalid input")
}

File: calc.py
def add(x, y):

   return x + y

def subtract(x, y):

   return x - y

def multiply(x, y):

   return x * y

def divide(x, y):

   return x / y

print("Select operation: ")
print("1. Add")
print("2. Subtract")
print("3. Multiply")
print("4. Divide")

choice = input("Select from above: ")

num1 = int(input("Enter the first number: "))
num2 = int(input("Enter the second number: "))

if choice == '1':
   print("The answer is: ", add(num1,num2))

elif choice == '2':
   print("The answer is: ", subtract(num1,num2))

elif choice == '3':
   print("The answer is: ", multiply(num1,num2))

elif choice == '4':
   print("The answer is: ", divide(num1,num2))
else:
   print("Invalid input")
